utilities/DumpUnitary/Driver.cs

```diff
@@ -7,54 +7,77 @@
 
 namespace Quantum.DumpUnitary
 {
-    class Driver
+    public class Driver
     {
-        static void Main(string[] args)
-        {
-            double eps = 1E-5;          // the square of the absolute value of the amplitude has to be less than or equal to eps to be considered 0
+        const double eps = 1E-5;          // the square of the absolute value of the amplitude has to be less than or equal to eps to be considered 0
 
+        public static void runSim(
```

Comment:
IntelliSense suggests to call this `RunSim`, with the first letter capitalized. I would actually call it `RunDumpUnitary` for better readability.